connector/oidc/oidc.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 
 	Scopes []string `json:"scopes"` // defaults to "profile" and "email"
 
+	HostedDomain string `json:"hostedDomain"`
```

Comment:
It might be nice to support an array of domains in the configuration and then match on "at least one" in HandleCallback?